test/object_store/test_basic.py

```diff
@@ -135,6 +135,11 @@ async def test_basic(test_tempdir, s3_server, ssl):
         # Sanity check that the path is constructed correctly
         assert os.path.exists(os.path.join(test_tempdir, 'data/system')), "Datadir is elsewhere"
 
+        desc = conn.execute(f"DESCRIBE KEYSPACE {ks}").one().create_statement
+        # The storage_opts wraps options with '{ <options> }' while the DESCRIBE
+        # does it like '{<options>}' so strip the corner brances and spaces for check
+        assert storage_opts.strip("{ }") in desc, "DESCRIBE generates unexpected storage options"
```

Comment:
I'm in favor of moving this test from cql-pytest/ to object_store/, but I'm less happy about the fact that in object_store/ everything is tested in one big test function `test_basic()`, and here you make it even bigger. I think the following approach can be better:
1. Create a fixture that sets up a table on s3.
2. Create a small test function that **only** checks DESCRIBE KEYSPACE

Anyway - this is for your consideration. I don't insist. There's nothing really incorrect with your approach either.

Reply:
I like the making the table as "fixture" suggestion. Meanwhile there's less pytest-ish generalization of this place in #15986 and the idea to keep the whole object_store test in dtests (#16006). So I'd prefer to postpone this particular change for later.

Reply:
I remembered. Currently object_store test cases do not use clusters provided by test.py, they create it on their own explicitly inside the test case function. Making ks.cf a fixture would require making scyllas be fixtures too, so it's much more work. Also I don't remember why @tchaikov didn't make it a fixture from the very beginning, there was some reason.

Reply:
the reason is i am lazy. back then, i was trying to migrate the test to pytest and focused on transforming the cluster (scylla instance) and s3 server into fixtures . they are the facilities provided by `test.py` . but put much less efforts on restructuring the test itself.

and yeah, i agree that the table creation/teardown is a great candidate of fixturization.

Reply:
> the reason is i am lazy

:rofl: